react_on_rails_pro/packages/node-renderer/tests/fixtures/bundle.js

```
@@ -1,3 +1,57 @@
+const { PassThrough } = require('stream');
+
 global.ReactOnRails = {
   dummy: { html: 'Dummy Object' },
+  
+  // Get or create async value promise
+  getAsyncValue: function() {
+    debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)